### DISCUSSION TITLE
How to add styles to current date in Date Range Pickers from react-aria-components

### DISCUSSION THREAD

**Discussion by RajaTheKing826** [adobe/react-spectrum#6935] [Q&A] [answered]
I'm developing a date range picker using react-aria-components but we are missing to style current date specially which is already in react spectrum date picker. 

Can you please tell us how to do it.

**Comment by reidbarber** [comment, marked as answer]
There is an `isToday` function in `@internationalized/date` that you can use: https://react-spectrum.adobe.com/internationalized/date/CalendarDate.html#comparison
You can pass the date for each cell into that.
